src/css/style.css

```diff
@@ -123,7 +123,74 @@ p{
 
 
 /* Carlo's section */
+#intro {
+  height: 60vh;
+  color: white;
+}
+
+#intro h1 {
+  font-size: 300%;
+}
+
+#intro p {
+  line-height: 1.5em;
+  font-size: 160%;
+}
+
+#ready-to-try {
+  background-image: url(../assets/CTA-banner-texture.jpg);
+  color: white;
+}
+
+#footer nav ul li a, #footer {
+  color: gray;
+}
+
+#footer i {
+  color: rgb(34,127,126);
+}
+
+#footer, #dev-workspace {
+  background-color: rgb(250,250,250);
+}
+
+div.col {
```

Comment:
You'll need to make a change here so that it only affects the sections you want. Try giving a more specific class name.

Reply:
you're right